src/gdt2gds.f

```diff
@@ -1,61 +1,49 @@
+C>    @file
+C>    @brief This routine converts grid information from a GRIB2 grid
+C>    to GRIB1 GDS info.
+C>    @author Stephen Gilbert @date 2003-06-17
+C>
+
+C>    This routine converts grid information from a GRIB2
+C>    Grid Description Section as well as its Grid Definition
+C>    Template to GRIB1 GDS info. In addition, a check is made
+C>    to determine if the grid is a NCEP predefined grid.
+C>
+C>    PROGRAM HISTORY LOG:
+C>    - 2003-06-17  Stephen Gilbert
+C>    - 2004-04-27  Stephen Gilbert Added support for gaussian grids.
+C>    - 2007-04-16  Boi Vuong Added Curvilinear Orthogonal grids.
+C>    - 2007-05-29  Boi Vuong Added Rotate Lat/Lon E-grid (203).
+C>
+C>    @param[in] igds Contains information read from the appropriate GRIB Grid
+C>    Definition Section 3 for the field being returned.
+C>    Must be dimensioned >= 5.
+C>    - igds(1) Source of grid definition (see Code Table 3.0)
+C>    - igds(2) Number of grid points in the defined grid.
+C>    - igds(3) Number of octets needed for each additional grid points definition.
+C>    Used to define number of points in each row (or column) for
+C>    non-regular grids. = 0, if using regular grid.
+C>    - igds(4) Interpretation of list for optional point definition. Code Table 3.11)
+C>    - igds(5) Grid Definition Template Number (Code Table 3.1)
+C>    @param[in] igdstmpl Grid Definition Template values for GDT 3.igds(5)
+C>    @param[in] idefnum The number of entries in array ideflist.
+C>    i.e. number of rows (or columns) for which optional grid points are defined.
+C>    @param[in] ideflist Optional integer array containing
+C>    the number of grid points contained in each row (or column).
+C>    @param[out] kgds GRIB1 GDS as described in w3fi63 format.
+C>    @param[out] igrid NCEP predefined GRIB1 grid number set to 255, if not NCEP grid.
+C>    @param[out] iret Error return value:
+C>    - 0 Successful
+C>    - 1 Unrecognized GRIB2 GDT number 3.igds(5)
+C>
+C>    @note LIST CAVEATS, OTHER HELPFUL HINTS OR INFORMATION
```

Comment:
This @note tag can be elimiated as it does not contain any information.

Reply:
yes